[website] New wave of open roles (#35240)
mui/material-ui: docs/src/pages/careers/accessibility-engineer.md

# Accessibility Engineer

> <p class="description">You will strengthen the core components team, by implementing best practices and guidlines related to the accessibility of the products.</p>

Reviewer: ```suggestion
<p class="description">You will strengthen the core components team by implementing best practices and guidelines related to the accessibility of the products.</p>
```

## Why we're hiring

Both our open-source community and our premium products are growing fast (x2-3 YoY).
We need talented people to keep that going!

We also need help to continue to improve the health of the open-source products: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally be a positive presence in the open-source community.

### Why this is interesting

Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month.
Hundreds of thousands of developers use MUI every month.

This means that the impact of your work in improving the accessibility of our products would literally help millions of people who use the products built with our libraries.

## About the role

### What you'll do on a day-to-day basis

Depending on the day, you'll:

- **Ensure the components are accessible.**

>   You would basically be an owner of the accessibility across all MUI products. It would be your responsibility to first verify the level of accessibility the components comply with and improve it over time.

Reviewer: ```suggestion
  You would basically be an owner of the accessibility across all MUI products. It would be your responsibility first to verify the level of accessibility the components comply with and improve it over time.
```

- **Respond to issues related to accessibility.**

>   Sometimes, users of our products create issues related to a11y, it would be your responsibility to answer them, by either providing best practices or fixing issues that they may have reported.

Reviewer: ```suggestion
  Sometimes, users of our products create issues related to a11y. It would be your responsibility to answer them by either providing best practices or fixing issues that they may have reported.
```

- **Intelligently apply accessibility testing tools, use assistive technology, and perform expert manual analysis of the MUI product lines.**
  Having a solid test framework that can help us make sure we stay compliant.
- **Lead the accessibility training across the company.**
  Basically, you would be the go-to person for anything related to accessibility, so mentoring others would be part of that deal.

## About you

### Skills you should have

- **A track record of accessibility experience with both consumer/enterprise products and assistive technologies, such as screen readers, eye-control, and other accessible technologies**.
  If you've been the accessibility champ in a previous company, pushing towards making sure the user interfaces can be used by disabled people, we want you in our team!
- **Hands-on knowledge of accessibility guidelines and standards (such as Section 508, WCAG, and EN 301 549) as well as technologies (such as UI Automation, IAccessible, ARIA, and related accessibility APIs)**

>   You would be the one that the team can go to for any a11t topics/questions.

Reviewer: ```suggestion
  You would be the one who the team can go to with any a11y topics/questions.
```

We released [MUI X](https://mui.com/x/), a collection of advanced components; [MUI Design kits](https://mui.com/design-kits/), the MUI components available for Figma, Sketch, etc.; and also host [Templates](https://mui.com/templates/), a set of pre-built UI kits.
We've seen rapid growth with all of them (2-3x per year) and we have more to come.
We are building [a low-code platform](https://mui.com/toolpad/) for developers to significantly increase the speed and lower the barrier for creating simple CRUD/dashboard UIs with MUI.

We are a fully distributed team, spread across multiple time zones.
We work mainly asynchronously, relying mostly on written collaboration.
Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).

## Why we're hiring

Accessibility is a strategic opportunty for MUI, across our products:

> 1. We are building MUI Base for the developers that are looking for a relatively lower-level API while keeping a strong baseline to build on top of that saves them time. For them, it's about having a better customizability experience (style & behavior). The most exigent users of MUI Base, the ones that are dictating the viral growth of if in the community, have strong a11y requirements, so we have to meet them for MUI Base to be successful.

Author: ```suggestion
1. We are building MUI Base for the developers that are looking for a relatively lower-level API while keeping a strong baseline to build on top of that saves them time. For them, it's about having a better customizability experience (style & behavior). The most exigent users of MUI Base, the ones that are dictating its viral growth in the community, have strong a11y requirements, so we have to meet them for MUI Base to be successful.
```

> 2. Accessibility is mandated by law. For example in the US, the Americans with Disabilities Act (ADA) is a US law enacted in 1990 to ensure people with disabilities are not discriminated against, in the EU, the European Accessibility Act (AEE) will soon be enacted. Our paid customers are increasingly asking about compliance with WCAG.

Reviewer: ```suggestion
2. Accessibility is mandated by law. For example, in the US, the Americans with Disabilities Act (ADA) is a US law enacted in 1990 to ensure people with disabilities are not discriminated against. In the EU, the European Accessibility Act (AEE) will soon be enacted. Our paid customers are increasingly asking about compliance with WCAG.
```

Both our open-source community and our premium products are growing fast (x2 YoY).
We need talented people to keep that going!

### Why this is interesting

Our solution empowers React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.

So, by improving the accessibility of our components, you will help millions of end-users who use [the products built with our UI library](https://trends.builtwith.com/framework/Material-UI).
You will also be responsible to train the developers using MUI's products to a11y. For example, there might be an opportunity to create an eslint plugin aware of Material UI APIs to catch a11y issues before they go to production.

## About the role

### What you'll do on a day-to-day basis

Depending on the day, you'll:

- **Ensure the components are accessible.**

>   You will be the owner of the accessibility across all MUI products. First, you will be responsible first to verify the level of accessibility the components comply with and implementing improvements over time.

Author: ```suggestion
  You will be the owner of the accessibility across all MUI products. First, you will be responsible to verify the level of accessibility the components comply with and then implement improvements over time.
```
